gamestonk_terminal/stocks/backtesting/bt_view.py

```diff
@@ -18,6 +20,75 @@
 np.seterr(divide="ignore")
 
 
+def display_whatif_scenario(
+    ticker: str,
+    num_shares_acquired: float,
+    date_shares_acquired: datetime,
+):
+    """Display what if scenario
+
+    Parameters
+    ----------
+    ticker: str
+        Ticker to check what if scenario
+    num_shares: float
+        Number of shares acquired
+    date_shares_acquired: str
+        Date at which the shares were acquired
+    """
+    data = yf.download(ticker, progress=False)
+
+    if not data.empty:
+        data = data["Adj Close"]
+
+    ipo_date = data.index[0]
+    last_date = data.index[-1]
+
+    if not date_shares_acquired:
+        date_shares_acquired = ipo_date
+        print("IPO date selected by default.")
```

Comment:
Maybe throw in what that is i.e.: IPO Date ( 10-30-1992 ) selected"

Reply:
The reason I didn't it's because the date is shown straight below, and having it repeated looked ugly 😋